pennylane/workflow/qnode.py

```diff
@@ -273,9 +273,30 @@ class QNode:
               rule for all supported quantum operation arguments, with finite-difference
               as a fallback.
 
-            * ``"hadamard"``: Use the analytic hadamard gradient test
-              rule for all supported quantum operation arguments. More info is in the documentation
-              :func:`qml.gradients.hadamard_grad <.gradients.hadamard_grad>`.
+            * ``"hadamard"``: Use the standard analytic hadamard gradient test rule for 
+            all supported quantum operation arguments. More info is in the documentation
+            for :func:`qml.gradients.hadamard_grad <.gradients.hadamard_grad>`.
+
+            * ``"reversed-hadamard"``: A variant of ``"hadamard"``, where the role of the 
+            observable being measured and the generators of the unitary operations in the
+            circuit are reversed; the generators are now the observables, and the Pauli
+            decomposition of the observables are now gates in the circuit. More 
+            information can be found in the documentation for 
+            :func:`qml.gradients.hadamard_grad <.gradients.hadamard_grad>` and the 
+            `original paper outlining this gradient method <https://arxiv.org/pdf/2408.05406>`__.
+
+            * ``"direct-hadamard"``: A variant of ``"hadamard"``, where the additional ancillary 

```

Comment:
TODO: change ancillary to auxiliary

Reply:
Resolved as transferred to https://github.com/PennyLaneAI/pennylane/pull/7198#discussion_r2029142798